unittests/tools/test_awssecurityhub_parser.py

```diff
@@ -101,3 +101,15 @@ def test_inspector_ecr(self):
             self.assertEqual("CVE-2023-2650 - openssl - Image: repo-os/sha256:af965ef68c78374a5f987fce98c0ddfa45801df2395bf012c50b863e65978d74", finding.title)
             self.assertIn("repo-os/sha256:af965ef68c78374a5f987fce98c0ddfa45801df2395bf012c50b863e65978d74", finding.impact)
             self.assertIn("Repository: repo-os", finding.impact)
+
+    def test_guardduty(self):
```

Comment:
Could you please extend the unittest here? Only checking for severity and active not make sure e.g. the mitigation or description are formatted right.

Reply:
Thank you for the hint. Done @quirinziessler